_includes/selector.html

```diff
@@ -252,15 +252,15 @@
                         </div>
                     </template>
                 </div>
-                <div class="options-section">
+                <div class="options-section" x-show="active_method != 'pip'">
```

Comment:
```suggestion
                <div class="options-section" x-show="active_method !== 'pip'">
```

Minor code suggestion for consistency with our other conditional statements.

Details on the difference between `!=` and `!==` are below:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Inequality
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality